modules/ROOT/pages/pricing.adoc

```diff
@@ -2,86 +2,50 @@
-* 4.2.0-20221027-6
-
-Mule 3 images version:
-
-* 3.9.5-20230424-2
-* 3.9.4-20230424-1
-* 3.9.3-20230424-1
-* 3.9.2-20230424-1
-* 3.9.1-20230424-1
-
-|=== 
-
-After your organization onboards to usage-based pricing, you must redeploy your apps with a supported version, either manually or during the monthly patching cycle. The monthly patching cycle is applicable only for CloudHub 1.0 and CloudHub 2.0 applications. RTF customers must be using RTF agent version 1.13.112 or 2.1.30 or newer. You must also manually redeploy Runtime Fabric applications (with a supported date-patch version) after your organization is onboarded to usage-based pricing.
-
-To xref:cloudhub-2::ch2-update-apps.adoc#roll-back-to-a-previous-successful-configuration[roll back] to a previous successful configuration of existing applications that use a Mule version that the usage-based pricing model does not support, choose a compliant version before deploying changes. After deployment, MuleSoft creates a new configuration with the previously chosen settings and a supported runtime.
+For Runtime Fabric, you must redeploy apps manually. You must redeploy Runtime Fabric apps in a timely manner to remain in compliance with your Anypoint Platform contract.
 
 == Metrics Definitions
 
 Usage-based pricing packages and their usage reports are based on your organization's use of allotted Mule components.
 
-For information about pricing concepts within Anypoint Platform, see xref:general::pricing-metrics.adoc[Usage and Pricing Metrics].
+To understand the technical aspects of metering and pricing concepts within Anypoint Platform, see xref:general::usage-metrics.adoc[Usage and Pricing Metrics] documentation.
 
 == Global Resource Pool Limits
 
-Root organizations provisioned with usage-based pricing receive a global pool of shared CPU and memory resources that is shared for deployments into CloudHub 1.0 and CloudHub 2.0 across all business groups and environments in the organization. This pool of resources is large enough to consume all of the purchased flows and messages without interruption. If an organization runs out of resources, actions that consume more resources such as deploying new applications or starting stopped applications are blocked until sufficient resources are available. 
+Root organizations provisioned with usage-based pricing receive a global pool of CPU and memory resources that are shared for CloudHub and CloudHub 2.0 deployments across all business groups and environments in the organization. This pool of resources is large enough to consume all the purchased flows and messages without interruption. 
+
+If an organization runs out of resources, actions that consume more resources, such as deploying new applications or starting stopped applications are blocked until sufficient resources are available. You can free up resources by stopping or deleting unused applications, or reducing the replica size or resources that an existing application consumes. 
```

Comment:
Do you need "stopped" here or is it implied?

Reply:
IMO, it's good to have.